financepy/utils/helpers.py

```diff
@@ -6,6 +6,8 @@
 import numpy as np
 from numba import njit, float64
 from typing import Union
+from prettytable import PrettyTable

```

Comment:
`prettytable` is imported globally, so it should be added to install requirements